src/views/DocumentationTopic.vue

```diff
@@ -39,30 +39,36 @@
               :api-changes-version="store.state.selectedAPIChangesVersion"
             >
               <template #default="slotProps">
-                <Navigator
-                  :parent-topic-identifiers="parentTopicIdentifiers"
-                  :technology="slotProps.technology || technology"
-                  :is-fetching="slotProps.isFetching"
-                  :error-fetching="slotProps.errorFetching"
-                  :api-changes="slotProps.apiChanges"
-                  :references="topicProps.references"
-                  :scrollLockID="scrollLockID"
-                  :breakpoint="breakpoint"
-                  @close="isSideNavOpen = false"
-                />
+                <transition name="delay-hiding">
+                  <Navigator
+                    v-show="isSideNavOpen || breakpoint === BreakpointName.large"
+                    :parent-topic-identifiers="parentTopicIdentifiers"
+                    :technology="slotProps.technology || technology"
+                    :is-fetching="slotProps.isFetching"
+                    :error-fetching="slotProps.errorFetching"
+                    :api-changes="slotProps.apiChanges"
+                    :references="topicProps.references"
+                    :scrollLockID="scrollLockID"
+                    :breakpoint="breakpoint"
+                    @close="isSideNavOpen = false"
+                  />
+                </transition>
               </template>
             </NavigatorDataProvider>
           </aside>
         </template>
-        <Topic
-          v-bind="topicProps"
-          :key="topicKey"
-          :objcPath="objcPath"
-          :swiftPath="swiftPath"
-          :isSymbolDeprecated="isSymbolDeprecated"
-          :isSymbolBeta="isSymbolBeta"
-          :languagePaths="languagePaths"
-        />
+        <transition name="delay-hiding">
```

Comment:
We should be applying an aria-hidden to everything except the navigator. This has been what we do for Modals up to now too, isnt that sufficient?

Reply:
No, it's not. Aria-hidden doesn't prevent keyboard navigation, it only prevents VO navigation.
I'll fill bugs for the modals too.

Reply:
what do you mean by keyboard navigation? Tabbing?

Reply:
Yes

Reply:
Does the FocusTrap work?

Reply:
That's a good point. I've been checking the FocusTrap class and actually `event.preventDefault();` inside `onFocus` function is not preventing from focusing outside the container. Is this a regression? Can you test it too?